internal/lib/migrator/migrator.go

```diff
@@ -0,0 +1,26 @@
+package migrator
+
+import (
+	"database/sql"
+
+	"github.com/pressly/goose"
+	"github.com/vadskev/urlshort/internal/config"
+)
+
+func Migrate(cfg *config.Config) error {
+	db, err := sql.Open("pgx", cfg.DataBase.DatabaseDSN)
+	if err != nil {
+		return err
+	}
+	defer func(db *sql.DB) {
+		err := db.Close()
+		if err != nil {
+			panic(err)
```

Comment:
```suggestion
			return err
```
все реакции на рожденную ошибку должны быть максимально высоко в иерархии кода.